lib/actions/appleMusic.js

```diff
@@ -155,15 +145,16 @@ function appleMusic(player, values) {
                 return player.coordinator.clearQueue()
                   .then(() => player.coordinator.addURIToQueue(queueTracks[0].uri, queueTracks[0].metadata, true, 1))
                   .then(() => {
-                      if (radioOn(player)) {
-                        return promise.then(() => player.coordinator.setAVTransport(queueURI,''));
+                      if (isRadioOrLineIn(player.coordinator.state.currentTrack.uri)) {
```

Comment:
What I'm trying to check is if what is currently playing is a "radio" stream and therefore there is a need to switch playing back to the queue.  An exception is thrown if you try to change back to the queue when it is already playing.

Given this, should I still check avTransportUri instead of the currentTrack.uri?